dist/ExtUtils-ParseXS/lib/perlxstut.pod

```diff
@@ -395,11 +395,27 @@ Edit the Makefile.PL file so that the corresponding line looks like this:
 Generate the Makefile and run make.  Change the test number in Mytest.t to
 "9" and add the following tests:
 
-	$i = -1.5; &Mytest::round($i); is( $i, -2.0 );
-	$i = -1.1; &Mytest::round($i); is( $i, -1.0 );
-	$i = 0.0; &Mytest::round($i);  is( $i,  0.0 );
-	$i = 0.5; &Mytest::round($i);  is( $i,  1.0 );
-	$i = 1.2; &Mytest::round($i);  is( $i,  1.0 );
+	my $i;
+
+	$i = -1.5
+	&Mytest::round($i)
```

Comment:
We should probably not show subroutine calls with `&` but that looks like it needs to be done more generally in this doc.

Reply:
I thought about it too, but decided against it until I read it more thoroughly.